docs/readmes/orc8r/deploy_build.md

```diff
@@ -92,22 +95,21 @@ export GITHUB_USERNAME=GITHUB_USERNAME
 export GITHUB_ACCESS_TOKEN=GITHUB_ACCESS_TOKEN
 ```
 
-Now run the package script. This script will package and publish the necessary
-helm charts to the `GITHUB_REPO`. The script expects a deployment type to be
-provided. The valid options are
+Next we'll run the package script. This script will package and publish the
+necessary helm charts to the `GITHUB_REPO`. The script expects a deployment
+type to be provided, which will determine which orc8r modules are deployed.
+The valid deployment type options are
+
 - `fwa`
```

Comment:
Since we don't have any page explaining what FWA, Federated FWA and all means, can you add a deployment page with brief information on what FWA means, federated FWA and all would mean.